### PR TITLE
AG-912 Fix navigation from circle popover

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts [hunk 1]
```diff
@@ -1087,13 +1087,12 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
   }
 
   navigateToConsistencyOfChange(data: any) {
-    const url = this.router.createUrlTree([
+    const baseURL = this.router.createUrlTree([
```
Comment (sagely1) [inline src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts:1090]: refactored this code for clarity

File: src/app/features/genes/components/gene-details/gene-details.component.spec.ts [hunk 2]
```diff
@@ -27,6 +27,13 @@ class ActivatedRouteStub {
     observer.next(convertToParamMap(paramMap));
     observer.complete();
   });
+  queryParams = new Observable((observer) => {
```
Comment (sagely1) [inline src/app/features/genes/components/gene-details/gene-details.component.spec.ts:30]: needed to add this mock due to child component now using queryParams

File: src/app/shared/helpers/app-helpers.ts [hunk 3]
```diff
@@ -2,4 +2,10 @@ export function getRandomInt(minInclusive: number, maxExclusive: number) {
   minInclusive = Math.ceil(minInclusive);
   maxExclusive = Math.floor(maxExclusive);
   return Math.floor(Math.random() * (maxExclusive - minInclusive + 1)) + minInclusive;
+}
+
+export function removeParenthesis(s: string) {
```
Comment (sagely1) [inline src/app/shared/helpers/app-helpers.ts:7]: created a helper function to remove parenthesis as window.open will remove parenthesis in the url and then the component cannot find the value without parenthesis.

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**